File: RestAPIs/UserAuthApi/Models/City.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace UserAuthApi.Models
{
    [Table("city")]
    public class City
    {
        [Key]
        [Column("city_id")]
        public int CityId { get; set; }

        [Column("state_id")]
        public int StateId { get; set; }

        [Column("city_name")]
        public string CityName { get; set; }
    }
}

File: User Services/freelancer/src/main/java/com/worksphere/freelancer/entity/City.java
package com.worksphere.freelancer.entity;

import jakarta.persistence.*;
import lombok.*;

@Entity
@Table(name = "city")
@Data
@NoArgsConstructor
@AllArgsConstructor
public class City {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "city_id")
    private Integer cityId;

    @Column(name = "city_name")
    private String cityName;

    @ManyToOne
    @JoinColumn(name = "state_id")
    private State state;
}
